storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -3488,31 +3489,31 @@ static dberr_t xb_load_tablespaces(void)
   err = srv_sys_space.check_file_spec(false, 0);
 
   if (err != DB_SUCCESS) {
-    msg("xtrabackup: could not find data files at the specified datadir\n");
+    xb::error() << "could not find data files at the specified datadir";
     return (DB_ERROR);
   }
 
   err =
       srv_sys_space.open_or_create(false, false, &sum_of_new_sizes, &flush_lsn);
 
   if (err != DB_SUCCESS) {
-    msg("xtrabackup: Could not open or create data files.\n"
-        "xtrabackup: If you tried to add new data files, and it "
-        "failed here,\n"
-        "xtrabackup: you should now edit innodb_data_file_path in "
-        "my.cnf back\n"
-        "xtrabackup: to what it was, and remove the new ibdata "
-        "files InnoDB created\n"
-        "xtrabackup: in this failed attempt. InnoDB only wrote "
-        "those files full of\n"
-        "xtrabackup: zeros, but did not yet use them in any way. "
-        "But be careful: do not\n"
-        "xtrabackup: remove old data files which contain your "
-        "precious data!\n");
+    xb::info() << "Could not open or create data files.\n"
+                  "xtrabackup: If you tried to add new data files, and it "
+                  "failed here,\n"
+                  "xtrabackup: you should now edit innodb_data_file_path in "
+                  "my.cnf back\n"
+                  "xtrabackup: to what it was, and remove the new ibdata "
+                  "files InnoDB created\n"
+                  "xtrabackup: in this failed attempt. InnoDB only wrote "
+                  "those files full of\n"
+                  "xtrabackup: zeros, but did not yet use them in any way. "
+                  "But be careful: do not\n"
+                  "xtrabackup: remove old data files which contain your "
+                  "precious data!\n";
```

Comment:
This one breaks the pattern of other messages:
- new line is a new `xb::info`
- remove `xtrabackup:` from line 2 to the end
- remove `\n`

Reply:
right, I rewrote to this
```
  xb::error() << "Could not open or create data files.";
      xb::error() << "If you tried to add new data files, and it "
                     "failed here,";
      xb::error() << "you should now edit innodb_data_file_path in "
                     "my.cnf back";
      xb::error() << "to what it was, and remove the new ibdata "
                     "files InnoDB created";
      xb::error() << "in this failed attempt. InnoDB only wrote "
                     "those files full of";
      xb::error() << "zeros, but did not yet use them in any way. "
                     "But be careful: do not";
      xb::error() << "remove old data files which contain your "
                     "precious data!";
      return (err);
    }

```

Reply:
it is an error, not info